docx/blkcntnr.py

```diff
@@ -23,6 +26,15 @@ def __init__(self, element, parent):
         super(BlockItemContainer, self).__init__(parent)
         self._element = element
 
+    def generate_numId(self):
+        """
+        Generate a unique numId value on this container.
+        """
+        while True:
+            numId = random.randint(0, 999999)
+            if not len(self._element.xpath("//w:numId[@w:val='%s']" % numId)):
```

Comment:
Do the quotes matter here? If so, shouldn't it be `'` for the outer, `"` for the inner?